app/public/map.js

```diff
@@ -194,6 +199,31 @@ var nazarethNuggets = (function () { // eslint-disable-line
   // on zoomend is good but not perfect, because can zoom multiple levels before this function will re-run
   mymap.on('zoomend', displayCorrectIcons)
 
+  function submitForm (e) { // eslint-disable-line
+    // get data from form fields
+    var form = e.target.parentNode.parentNode
+    var inputs = form.querySelectorAll('.add-form-input')
+    var formData = {}
+    inputs.forEach(function (input) {
+      formData[input.name] = input.value
```

Comment:
yeh I guess... or we merge your form and then we add the name attributes to make this branch work